packages/core/src/App/Containers/RedirectToLoginModal/redirect-to-login-modal.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { connect } from 'Stores/connect';
+import { Dialog, Text } from '@deriv/components';
+import { Localize, getLanguage } from '@deriv/translations';
+import { redirectToLogin } from '@deriv/shared';
+
+const ModalHeader = ({ header }) => {
+    switch (header) {
+        case 'password_changed':
+            return (
+                <Text as='p' weight='bold' className='reset-password__heading'>
+                    <Localize i18n_default_text='Your password has been changed' />
+                </Text>
+            );
+        default:
+            return '';
+    }
+};
+
+const RedirectToLoginModal = ({ is_logged_in, logout, disableApp, enableApp, is_loading }) => {
+    const [is_visible, setVisible] = React.useState(false);
+    const url_params = new URLSearchParams(useLocation().search);
+    const header = url_params.get('header');
+
+    const showModal = () => {
+        setVisible(true);
+        redirectToLogin(false, getLanguage(), true);
```

Comment:
If you applied the suggested change to `redirectToLogin` then don't forget to revert 3rd argument to `false`. 🙏🏻